packages/services-context/src/index.ts

```diff
@@ -45,12 +44,10 @@ export class Services implements ServicesInterface {
    *
    * @returns completed AppParameters from remote or storage
    */
-  private async getParams(querier: RootQuerier): Promise<AppParameters> {
-    // try to read params from storage
-    const storedParams = await localExtStorage
-      .get('params')
-      .then(j => j && AppParameters.fromJson(j))
-      .catch(() => undefined);
+  private async getParams(indexedDb: IndexedDb, querier: RootQuerier): Promise<AppParameters> {
+    // try to read params from idb
+    const storedParams = await indexedDb.getAppParams();
```

Comment:
The function code comment seems to imply this is a fallback, probably needs updating:

```
   * Attempt to fetch the AppParameters from the remote fullnode, or fall back
   * to the local storage. Will throw to abort if the remote node reports an
```